plugins/vli/fu-vli-common.h

```diff
@@ -37,6 +37,7 @@ typedef enum {
 	FU_VLI_DEVICE_KIND_MSP430		= 0xf430, /* guessed */
 	FU_VLI_DEVICE_KIND_PS186		= 0xf186, /* guessed */
 	FU_VLI_DEVICE_KIND_RTD21XX		= 0xff00, /* guessed */
+	FU_VLI_DEVICE_KIND_HYPER		= 0x0817,
```

Comment:
this is just an enum value, I'll fix it up to 0xff01 for you.

Reply:
Thank you!